docs/getting-started/integration_rewards.md

```diff
@@ -8,6 +8,15 @@ permalink: /getting-started/integration-rewards/
 
 # Integration rewards
 
-We encourage you to apply for integration rewards (upto **$20,000**) once your project is
-successfully integrated with OSS-Fuzz. Please see the details in our blog post
+We encourage you to apply for integration rewards (up to **$20,000**) once your project
+is successfully integrated with OSS-Fuzz. Please see the details in our blog post
 [here](https://opensource.googleblog.com/2017/05/oss-fuzz-five-months-later-and.html).
+
+Note that the biggest rewards are issued to projects that satisfy both of the following
```

Comment:
biggest or just "higher" ?

Reply:
I was trying to refer to the exact top amount ($20,000)

Reply:
ah ok, lgtm. "top rewards" sounds slightly better, upto you.

Reply:
I like "top rewards" too, thanks! fixing